<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSettings</class>
 <widget class="QDialog" name="MainSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AvisionR - Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_BusinessAddress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab
{
    background-color: #686868;

}

QTabWidget::pane
{
 border: 0px;
 top: 0.1em;
 border-image: url(tabBG.png);
     border-width: 1px;
     border-style: solid;
     border-color: grey;
}

color: black;
border: 1px solid #686868;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Address">
    <attribute name="title">
     <string>Address</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>312</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>City:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_storeCity">
        <property name="minimumSize">
         <size>
          <width>281</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #686868;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>178</y>
       <width>309</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_storeState">
        <property name="minimumSize">
         <size>
          <width>271</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #686868;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>122</y>
       <width>313</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Space/Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_StoreAddress2">
        <property name="minimumSize">
         <size>
          <width>231</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #686868;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>103</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>AvisonR Address</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>94</y>
       <width>313</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Address Line 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_storeAddress1">
        <property name="minimumSize">
         <size>
          <width>231</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #686868;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_save">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>262</y>
       <width>172</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>283</width>
       <height>23</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>101</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Phone Number:        (</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_PhoneArea">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #686868;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>)  -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_PhoneMiddle">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #686868;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string> -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_PhoneRight">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #686868;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>207</y>
       <width>311</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string>Zip Code:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_StoreZip">
        <property name="styleSheet">
         <string notr="true">border: 1px solid #686868;</string>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget_2</zorder>
    <zorder>layoutWidget_3</zorder>
    <zorder>layoutWidget_4</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>label</zorder>
    <zorder>commandLinkButton_save</zorder>
    <zorder>label_2</zorder>
    <zorder>lineEdit_phone</zorder>
    <zorder>layoutWidget_3</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
